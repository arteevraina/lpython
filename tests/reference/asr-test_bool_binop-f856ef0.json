{
    "basename": "asr-test_bool_binop-f856ef0",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_bool_binop.py",
    "infile_hash": "7c44fe2915b3871b3b85edf3ab129d87a75fa4b9acbb597ae801daf9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_bool_binop-f856ef0.stdout",
<<<<<<< HEAD
    "stdout_hash": "c5e0ecc9ff081961a546fab314b4f0b4ef23f28cf9a79fa0738832e8",
=======
    "stdout_hash": "17f23bdb23ec6588306416cfc406f24efad6516162c5ef0df1d3d82b",
>>>>>>> main
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}