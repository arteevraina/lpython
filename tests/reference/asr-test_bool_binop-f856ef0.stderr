warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
  --> tests/../integration_tests/test_bool_binop.py:14:13
   |
14 |     i = i32(True // True)
   |             ^^^^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
